packages/client/src/v2-events/features/events/components/Review.tsx

```diff
@@ -378,31 +388,45 @@ function FormReview({
               >
                 <ListReview id={'Section_' + page.id}>
                   {displayedFields.map(
-                    ({ id, label, errorDisplay, valueDisplay }) => (
-                      <ListReview.Row
-                        key={id}
-                        actions={
-                          !readonlyMode && (
-                            <Link
-                              data-testid={`change-button-${id}`}
-                              onClick={(e) => {
-                                e.stopPropagation()
-
-                                onEdit({
-                                  pageId: page.id,
-                                  fieldId: id
-                                })
-                              }}
-                            >
-                              {intl.formatMessage(reviewMessages.changeButton)}
-                            </Link>
-                          )
-                        }
-                        id={id}
-                        label={intl.formatMessage(label)}
-                        value={errorDisplay || valueDisplay}
-                      />
-                    )
+                    ({
+                      id,
+                      label,
+                      errorDisplay,
+                      valueDisplay,
+                      isCorrectable
+                    }) => {
+                      const shouldShowEditLink =
+                        !readonlyMode &&
+                        (!isCorrection || isCorrectable !== false)
```

Comment:
This makes the condition more readable.
```suggestion
                      const shouldShowEditLink =
                        !(readonlyMode ||
                        (isCorrection && isCorrectable === false))
```